pennylane/transforms/add_noise.py

```diff
@@ -74,7 +78,14 @@ def add_noise(tape, noise_model, level=None):
         def noise2(op, **kwargs):
             qml.ThermalRelaxationError(op.parameters[0] * 0.5, kwargs["t1"],  kwargs["t2"], 0.6, op.wires)
 
-        noise_model = qml.NoiseModel({fcond1: noise1, fcond2: noise2}, t1=2.0, t2=0.2)
+        fcond3 = qml.noise.meas_eq(qml.expval) & qml.noise.wires_in([0, 1]),
+        noise3 = qml.noise.partial_wires(qml.PhaseFlip, 0.2)
+
+        noise_model = qml.NoiseModel(
+            {fcond1: noise1, fcond2: noise2},
+            meas = {fcond3: noise3},
+            t1=2.0, t2=0.2
```

Comment:
The example is not working when adding the "meas" dictionary

Reply:
There was an extra comma, so should be fixed.